package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "appbase-js",
-	"version": "2.3.0",
+	"version": "2.3.1",
```

Comment:
Would this be a patch release or something else?

I hope it doesn't break things for those who depends on status 400 inside `.on('data')`.

cc @divyanshu013 :yum:

Reply:
Please do a major release 🙏